packages/framework/src/Model/Transport/TransportPriceCalculation.php

```diff
@@ -50,22 +50,25 @@ public function calculatePrice(
             return Price::zero();
         }
 
-        return $this->calculateIndependentPrice($transport, $currency);
+        return $this->calculateIndependentPrice($transport, $currency, $domainId);
     }
 
     /**
-     * @param \Shopsys\FrameworkBundle\Model\Transport\Transport $transport
+     * @param \Shopsys\FrameworkBundle\Model\Transport\Transport|null $transport
      * @param \Shopsys\FrameworkBundle\Model\Pricing\Currency\Currency $currency
+     * @param int $domainId
      * @return \Shopsys\FrameworkBundle\Model\Pricing\Price
      */
-    public function calculateIndependentPrice(
-        Transport $transport,
-        Currency $currency
-    ): Price {
+    public function calculateIndependentPrice(?Transport $transport, Currency $currency, int $domainId): Price
```

Comment:
How can you calculate price for not existing Transport? That is weird

Reply:
Not answered

Reply:
Like a previously answer. I can add null condition into facade and not have nullable entity in this method.